pom.xml

```diff
@@ -131,6 +131,12 @@
       <artifactId>domain-models-runtime</artifactId>
       <version>${raml-module-builder.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I know there was a separate PR, but was there a separate JIRA for the introduction of DI support?  Either way, it might be worth mentioning this in the release notes

Reply:
wait - is this just now being added as part of the release PR?  I thought it was added earlier in another PR?